### PR TITLE
Feature/game blocks

### PR DESCRIPTION
**Pull request by olhazamlynska**

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { StyledFotter, StyledText } from './Footer.styled';
+
+const Footer = () => {
+  return (
+    <StyledFotter>
+      <MainContainer>
+        <StyledText>Join us on our site </StyledText>
+        <a href="https://chingu-voyages.github.io/v44-tier2-team-15/">
+          Here
+          {/* <img src={logo} alt="Logo" width="76" height="22" /> */}
```
Comment (wayungi) [inline src/components/Footer/Footer.jsx:11]: - [ ] I kindly suggest that you delete all unused pieces of code left as comments.  Commented code left as comments make the code base messy 😄

File: src/components/GameControls/GameControls.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const GameControlls = () => {
+  return (
+    <div>
+      GameControlls
+      <button>Battle</button>
+      <button>Battle</button>
```
Comment (wayungi) [inline src/components/GameControls/GameControls.jsx:6]: - [ ] I think we need a `battle button` &  a `stop button`. Here I see 2 battle buttons. Do u plan to edit this further 👍🏾

File: src/pages/MainPage/MainPage.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <MainContainer>
+      <h1>Logic Warriors</h1>
+      <p>
+        Welcome to Logic Warriors! Click <Link to={'/game'}>GAME</Link> to
+        battle!
+      </p>
+    </MainContainer>
+  );
+};
+
```
Comment (wayungi) [inline src/pages/MainPage/MainPage.jsx:14]: - [ ] Great work @olhazamlynska , This is amazing 👏🏾

**Review by wayungi — APPROVED** [review]
Hi @olhazamlynska , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Great work ✔️ 
- [ ] Many features implemented ✔️ 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (olhazamlynska) [reply src/components/Footer/Footer.jsx:11]: Yes, i will delete it

File: src/components/GameControls/GameControls.jsx (hunk 2 above)
Comment (olhazamlynska) [reply src/components/GameControls/GameControls.jsx:6]: We need a button, where text changes from 'Buttle' to 'Stop'  and reverse

File: src/pages/MainPage/MainPage.jsx (hunk 3 above)
Comment (olhazamlynska) [reply src/pages/MainPage/MainPage.jsx:14]: Thanks

**Pull request merged**